examples/discordjs/src/index.ts

```diff
@@ -0,0 +1,18 @@
+import { Client, Message } from 'discord.js';
+const { DISCORD_TOKEN } = process.env; // The TOKEN from railway
+
+const client = new Client();
+
+client.on('ready', () => {
+	console.log(`Logged in as ${client.user?.tag}!`);
+});
+
+client.on('message', (msg: Message) => {
+	if (msg.content === 'ping') {
+		msg.reply('Pong 🏓');
+	} else if (msg.content === 'hello') {
+		msg.reply('Choo choo! 🚅');
+	}
+});
+
+client.login(DISCORD_TOKEN);
```

Comment:
Omit this too.

Reply:
Replace it with `client.login();`.

Reply:
Ya I wasn't sure this was part of v13 or v12 so I decided to keep it that way. However I've went ahead & confirmed that it is able to check token from env. Thanks for your feedback. I'll try to change this as soon as possible. 👍